dxpr/dxpr_marketing_cms
# Fixed #48: Removed comment module and related configurations

**Description:** For users installing DXPR Marketing CMS, there is an issue related to a missing relationship in the "Recent Comments" view. This occurs because there are no content types with a "Comment" field. Initially, the solution I considered was to introduce a hidden comment field, which was implemented in this pull request: https://github.com/dxpr/dxpr_marketing_cms/pull/49. However, upon reviewing the feedback in the discussion here: https://github.com/dxpr/dxpr_marketing_cms/pull/49#issuecomment-1667704105, I decided to remove the Comment module from the list of installed modules.

Moving forward, it will be the responsibility of users to install the Comment module themselves, giving them control over the existence of the Comment field. For users who are already experiencing the issue, we will provide a text description outlining the necessary steps to resolve it.

**Context:** 
1. https://github.com/dxpr/dxpr_marketing_cms/issues/48
2. It was reported by @jjroelofs in Slack (private messages)
3. https://www.drupal.org/project/dxpr_builder/issues/3378334 (2nd issue)

**Screenshots:**
![image](https://github.com/dxpr/dxpr_marketing_cms/assets/29509266/c99a67b5-84c3-4e29-b358-4343c3cdb9f2)

**Testing instructions:**
1. Pull DXPR Marketing CMS with this branch or with the branch that contains these changes.
2. Follow normal installation process described in DXPR Marketing CMS documentation.
3. After website installation - check, if Comment module is uninstalled.
4. Verify that issue no longer appear after cache rebuild.

**Other notes:**
1. There are issues during DXPR Marketing CMS installation which are not related to this fix. They should be treated separately.
